ios/RNCPagerView.h

```diff
@@ -2,22 +2,29 @@
 #import <React/RCTShadowView.h>
 #import <React/UIView+React.h>
 #import <UIKit/UIKit.h>
+#import "UIView+isHorizontalRtlLayout.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface RNCPagerView: UIView
+@interface RNCPagerView: UIView <RtlLayoutProtocol>
 
-- (instancetype)initWithEventDispatcher:(id<RCTEventDispatcherProtocol> )eventDispatcher;
+- (instancetype)initWithEventDispatcher:(RCTEventDispatcher *)eventDispatcher;
 
 @property(nonatomic) NSInteger initialPage;
-@property(nonatomic) NSString* orientation;
+@property(nonatomic) NSInteger lastReportedIndex;
+@property(nonatomic) NSInteger destinationIndex;
+@property(nonatomic) NSInteger currentIndex;
+@property(nonatomic) NSInteger pageMargin;
+@property(nonatomic, readonly) BOOL scrollEnabled;
```

Comment:
Why is this now exposed as public API? This should be hidden in @implementation as before

Reply:
to be addressed in followup PR